Lesson2/src/Homework.java

```diff
@@ -0,0 +1,103 @@
+import java.util.Scanner;
+
+public class Homework {
+
+    public static void main(String[] args) {
+        System.out.println(sum(10, 20));
+        System.out.println(sum(Integer.MAX_VALUE, Integer.MAX_VALUE));
+        System.out.println(max(50, 49));
+        System.out.println(average(new int[]{1, 2, 3, 4, 5}));
+        System.out.println(max(new int[]{1, 2, 3, 4, 5, 100, 99}));
+        System.out.println("Hypotenuse is: " + calculateHypotenuse());
+    }
+
+    /**
+     * 1. Method should return sum of "a" and "b"
+     * 2. (optional) if sum of a and b is more then int can store -- return -1;
+     **/
+    public static int sum(int a, int b) {
```

Comment:
you can clean up some parentheses here

 if (((long) a + b) > Integer.MAX_VALUE) ->  if ((long) a + b > Integer.MAX_VALUE)
 
return (a + b); ->   return a + b;